jmix-flowui/flowui/src/main/java/io/jmix/flowui/sys/JmixVaadinServletContextInitializer.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2022 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.flowui.sys;
+
+import com.vaadin.flow.spring.VaadinServletContextInitializer;
+import org.springframework.context.ApplicationContext;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.ServletException;
+
+public class JmixVaadinServletContextInitializer extends VaadinServletContextInitializer {
```

Comment:
Let's add javadoc that explains that this is a workaround for the issue, like you did in JmixTempJarManifestUtils class